styles.css

```diff
@@ -1,3 +1,1317 @@
+    background-repeat: no-repeat;
+    cursor: pointer;
+    background-position: center;
+}
+
+.details-button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 4px;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+    padding: 12px 36px 12px 36px;
+    border: none;
+    cursor: pointer;
+    background-color: transparent;
+    border: 1px solid #21E786;
+    color: #fff;
+}
+
+.whitelist,
+.read-more {
+    color: #fff;
+    background-color: transparent;
+    border: 2px solid #21E786;
+}
+
+.discord {
+    color: #fff;
+    background-color: #141B22;
+}
+
+.connect,
+.connect-wallet,
+.buy,
+.joc-whitelist {
+    color: #141B22;
+    background: #21E786;
+}
+
+.join-discord,
+.join-twitter {
+    color: #fff;
+    background-color: transparent;
+    border: 2px solid #888B8E;
+}
+
+.joc-discord {
+    color: #fff;
+    background-color: #5865F2;
+}
+
+.breadcrumbs-nav a {
+    display: flex;
+    font-size: 14px;
+    color: #888B8E;
+    font-weight: 400;
+    line-height: 20px;
+}
+.breadcrumbs-nav a:first-of-type::after{
```

Comment:
Here, I would apply styles to all elements except the last one, as breadcrumbs may have more than two items.

Breadcrumbs often contain multiple items, and the last item typically represents the current page and doesn't need a link or specific styling. By targeting all items except the last one, you ensure the breadcrumbs are styled correctly while maintaining flexibility for longer breadcrumb trails.